modules/virt-configuring-masquerade-mode-dual-stack.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc
+
+[id="virt-configuring-masquerade-mode-dual-stack_{context}"]
+= Configuring masquerade mode with dual-stack (IPv4 and IPv6)
+
+You can configure virtual machines to use both IPv6 and IPv4 on the default Pod Network. The IPv6 network address must be statically set in the virtual machine.
+
+Once configured, incoming and outgoing traffic for the virtual machine is routed to both the IPv4 address and the unique IPv6 address of the virt-launcher Pod.
+The virt-launcher Pod then routes the IPv4 traffic to the DHCP address of the virtual machine, and the IPv6 traffic to the statically set IPv6 address of the virtual machine.
+
+This procedure can be applied to new or existing virtual machines in the cluster.
+
+.Prerequisites
+
+* The underlying network for the cluster must be connected with IPv6.
+
+* The OpenShift cluster must be configured to use IPv6.
+
+.Procedure
+
+. Edit the virtual machine configuration to add an interface with `masquerade` and configure the IPv6 IP address, default gateway, and DNS record using cloud-init.
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachine
+metadata:
+  name: example-vm-ipv6
+...
+          interfaces:
+            - name: red
+              masquerade: {} <1>
+              ports:
+                - port: 80 <2>
+      networks:
+      - name: red
+        pod: {}
+      volumes:
+      - cloudInitNoCloud:
+          networkData: |
+            version: 2
+            ethernets:
+              eth0:
+                dhcp4: true
+                addresses: [ fd10:0:2::2/120 ] <3>
+                gateway6: fd10:0:2::1 <4>
+----
+<1> Connect using masquerade mode.
+<2> Allows incoming traffic on port 80 to the virtual machine.
+<3> The IPv6 address `fd10:0:2::2/120` must be used.
```

Comment:
Please also indicate that this speciifc address is always the one to be used for every VM created, no matter how many VMs there are in the cluster.

Reply:
I think "you must use X." gives a stronger message than "you must X for every Y created, no matter how many Ys you have in the cluster."